merlin_models/tf/block/inputs.py

```diff
@@ -52,52 +52,67 @@ def InputBlock(
     seq_aggregator: Block = SequenceAggregator(SequenceAggregation.MEAN),
     **kwargs,
 ) -> Block:
-    """Input Block to process categorical and continuous input features.
+    """The entry block of the model to process input features from a schema.
+
+    This function creates continuous and embedding layers, and connects them via `ParallelBlock`.
```

Comment:
Maybe we could add a short description of the sequential case (seq=True)? In this case, the features are split into context and sparse groups, then two InputBlocks are generated, and finally, these blocks are connected via a `ParallelBlock`.